funcx_endpoint/funcx_endpoint/endpoint/interchange.py

```diff
@@ -218,6 +218,7 @@ def load_config(self):
                 if not executor.endpoint_id == self.endpoint_id:
                     raise Exception('InconsistentEndpointId')
             self.executors[executor.label] = executor
+            executor.run_dir = self.logdir
```

Comment:
Since we could have multiple executors, try doing this instead:
`[x.run_dir = self.logdir for x in self.executors]`

Reply:
Ignore this. This is already in a loop.